README.md

```diff
@@ -68,6 +68,7 @@ Full list of the options
 
 | NAME                                | DESCRIPTION                                                    | TYPE     | DEFAULT               | OPTIONS                                  |
 | ----------------------------------- | -------------------------------------------------------------- | -------- | --------------------- | ---------------------------------------- |
+| `github-api-url`                    | The Github API endpoint. Override for Github Enterprise usage. | `string` | `true`                | `https://api.github.com`                 |
```

Comment:
At least for REST endpoint, this option name is good 👍 ref: https://github.com/actions/create-github-app-token/pull/88

---

```suggestion
| `github-api-url`                    | The Github API endpoint. Override for Github Enterprise usage. | `string` | `${{ github.api_url }}`                | `https://api.github.com`                 |
```

? (sorry for unformatting)